utils/Rector/UnderscoreToCamelCaseVariableNameRector.php

```diff
@@ -113,13 +113,16 @@ public function refactor(Node $node): ?Node
 
     private function updateDocblock(Variable $variable, string $variableName, string $camelCaseName): void
     {
-        $parentClassMethodOrFunction = $this->betterNodeFinder->findParentByTypes($variable, [ClassMethod::class, Function_::class]);
+        $parentFunctionLike = $this->betterNodeFinder->findParentType($variable, ClassMethod::class);
```

Comment:
`findParentType()` just removed at rector-src dev-main at:

- https://github.com/rectorphp/rector-src/pull/4438

I will refactor this rule to get Variable from: `FileWithoutNamespace`, `Namespace`, `ClassMethod`, `Function_`, `Closure`.